### DISCUSSION TITLE
新的flutter和蓝灯7.50通用账号吗

### DISCUSSION THREAD

**Discussion by hunterooo1234** [getlantern/lantern-forum-cn#520] [Q&A]
如题  我下了新的flutter 8.0 可以连接 但是无法登录我蓝灯的账号 原账号还有1年使用期 请问这个和蓝灯7.8.5是通用的吗 如果是如何解决问题

**Comment by MaxMKMax** [comment]
@hunterooo1234 您好. 请您等一等. 我帮您问一下.

**Comment by MaxMKMax** [comment]
@hunterooo1234 您好. 请您提供我您的账号邮箱和支付单据. 我看我们这里能怎么帮您处理这个问题. 谢谢!
